telephony/android_modem.c

```diff
@@ -3218,6 +3258,8 @@ handleDefinePDPContext( const char*  cmd, AModem  modem )
         amodem_begin_line( modem );
         amodem_add_line( modem, "+CGDCONT: (1-%d),\"IP\",,,(0-2),(0-4)",
                          MAX_DATA_CONTEXTS );
+        amodem_add_line( modem, "+CGDCONT: (1-%d),\"IPV6\",,,(0-2),(0-4)",
+                         MAX_DATA_CONTEXTS );
```

Comment:
Nit: Since now we also support IPV6, please help to update the comments in line#3255.